SmartDeviceLink_Example/Classes/ProxyManager.m

```diff
@@ -324,7 +333,7 @@ - (void)prepareRemoteSystem {
         dispatch_group_leave(dataDispatchGroup);
 
         if (success == NO) {
-            NSLog(@"Something went wrong, image could not upload: %@", error);
+            SDLLogW(@"Something went wrong, image could not upload: %@", error);
```

Comment:
Why issue a warning instead of an error? Seems like an image not uploading would be an example of an error?